### ISSUE TITLE
Training with Resnet50

### ISSUE THREAD

**Issue by SonDaoDuy** [akshitac8/BiAM#12]
Thank you for your great work and the code release, I've learned a lot from your work.
I just wonder if you tried BiAM with different backbone networks such as Resnet or VIT? 
I try to train BiAM with features extracted from Resnet50, but it is very easy to get 'nan' loss value after several training epochs. Do you have any suggestions to fix this?

**Comment by akshitac8** [comment]
@SonDaoDuy Please check the data augmentations you are using while extracting the features. You can also check if the features are normalized or not. Lastly, please try to change the hyper-parameters of BiAM while using different backbone features.
